cogent/stack-8.4.3.yaml

```diff
@@ -0,0 +1,14 @@
+resolver: lts-12.14             # ghc-8.4.3; 2018-10-21
+packages:
+  - '.'
+  - '../isa-parser'
+
+flags:
+  language-c-quote:
+    full-haskell-antiquotes: false
+
+extra-package-dbs: []
+extra-deps:
+  - sbv-7.12                    # for the `array' branches
```

Comment:
A question: on my branch, I use `sbv > 8`. Which version of `sbv`  can I have via `extra-deps`?

Reply:
This is only to get the pins right given the current builtin-arrays code. In theory, you can have whatever version you like. I believe sbv-8.x works fine, although I vaguely recall running into issues with crackNum, a sbv dependency, on GHC 8.2; I'd need to test that again to check, and I'd want to make sure stuff still compiles happily in general anyway.